src/pages/_document.tsx

```diff
@@ -1,14 +1,23 @@
-// Copyright (C) SW360 Frontend Authors (see <https://github.com/eclipse-sw360/sw360-frontend/blob/main/NOTICE>)
-
-// SPDX-License-Identifier: EPL-2.0
-// License-Filename: LICENSE
+/*
+ * SPDX-FileCopyrightText: (C) TOSHIBA CORPORATION, 2023.
+ * SPDX-FileCopyrightText: (C) Toshiba Software Development (Vietnam) Co., Ltd., 2023.
+ * SPDX-License-Identifier: EPL-2.0
+ * License-Filename: LICENSE
+ */
 
 import { Html, Head, Main, NextScript } from 'next/document'
+import Script from 'next/script'
 
 export default function Document() {
   return (
     <Html lang="en">
-      <Head />
+      <Head>
+      <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+              rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+              crossOrigin="anonymous" />
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css"></link>
+      </Head>
+
```

Comment:
Don't do this way.
Use this or similar class:
https://www.npmjs.com/package/react-bootstrap-icons